tests/test_context.py

```diff
@@ -0,0 +1,57 @@
+# Source pytest module
+# Copyright (C) 2020  Ngo Ngoc Duc Huy
+#
+# This file is part of palace.
+#
+# palace is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License,
+# or (at your option) any later version.
+#
+# palace is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with palace.  If not, see <https://www.gnu.org/licenses/>.
+
+"""This pytest module tries to test the correctness of the class Context."""
+
+from palace import Context, Device
+
+
+def test_with_context():
+    """Test if `with` can be used to start a context
+    and is destroyed properly.
+    """
+    dev = Device()
```

Comment:
You can use the device fixture, e.g. `def test_with_context(device)`.  I think that'd be a good idea since device is not part of the test.